Gulpfile.js

```diff
@@ -1,35 +1,6 @@
-var babel                = require('babel-core');
-var gulp                 = require('gulp');
-var gulpBabel            = require('gulp-babel');
-var less                 = require('gulp-less');
-var filter               = require('gulp-filter');
-var git                  = require('gulp-git');
-var globby               = require('globby');
-var qunitHarness         = require('gulp-qunit-harness');
-var mocha                = require('gulp-mocha');
-var mustache             = require('gulp-mustache');
-var rename               = require('gulp-rename');
-var webmake              = require('gulp-webmake');
-var util                 = require('gulp-util');
-var gulpif               = require('gulp-if');
-var uglify               = require('gulp-uglify');
-var ll                   = require('gulp-ll');
-var del                  = require('del');
-var fs                   = require('fs');
-var path                 = require('path');
-var opn                  = require('opn');
-var connect              = require('connect');
-var spawn                = require('cross-spawn');
-var serveStatic          = require('serve-static');
-var Promise              = require('pinkie');
-var markdownlint         = require('markdownlint');
-var prompt               = require('gulp-prompt');
-var nodeVer              = require('node-version');
-var functionalTestConfig = require('./test/functional/config');
-var assignIn             = require('lodash').assignIn;
-var runSequence          = require('run-sequence');
-var yaml                 = require('js-yaml');
-
+var ll = require('gulp-ll');
```

Comment:
It would be nice to separate task files by directories, e.g.: `lint`, `website`, `functional-tests`, `server-tests`, etc.